### DISCUSSION TITLE
QT6

### DISCUSSION THREAD

**Discussion by IHateYourCode** [texstudio-org/texstudio#1823] [Q&A] [answered]
I was wondering how far the QT6 branch is
and what the roadmap in general looks like.

**Comment by sunderme** [comment, marked as answer]
QT6 is supported since 4.0.0

**Reply by IHateYourCode** [reply]
And when will the QT6 branch be merged into master?

**Reply by sunderme** [reply]
That has already happened.

**Reply by IHateYourCode** [reply]
Oh, I see, so then I assume all the documentation referencing qt4/5 being used is also just outdated?

**Reply by sunderme** [reply]
It is Qt5/6 now.
Qt4 was abandoned with 3.x

**Reply by IHateYourCode** [reply]
Right, ok, good to know.
Thank you for the information.
